contentcuration/contentcuration/frontend/channelList/views/KCard/BaseCard.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div
+    class="card"
+    tabindex="0"
+    @focus="cardFocus"
+    @hover="cardHover"
+  >
+    <div
+      :class="$computedClass({ ':focus': $coreOutline })"
+    >
+      <li class="remove-list-style">
+        <component
+          :is="'h' + headingLevel"
+          v-if="title !== null"
```

Comment:
- I don't think conditional display is necessary since heading level is a required prop.  If you need to add conditional display, i think it would be best suited for `TextTuncator` component that holds the `title`. I'm not entirely sure but I think hiding the `h<2-6>` component would hinder click event propagation since we are looking at having the entire card responding to clicks. This we can clarify with Misha
- It's also worth considering using a computed prop for the `:is` value

Reply:
Okay-Thanks!